client/src/visualiser-src/binary-search-tree-visualiser/animation-producer/BSTAnimationProducer.ts

```diff
@@ -15,40 +15,32 @@ export default class BSTAnimationProducer extends AnimationProducer {
   }
 
   public flashNode(node: Node): void {
-    this.allRunners.push([
-      node.nodeTarget
-        .animate(500)
-        .attr({
-          fill: '#4beb9b',
-          stroke: '#4beb9b',
-        }),
-      node.textTarget
-        .animate(500)
-        .attr({
-          fill: '#ffffff',
-        }),
+    this.addAnimation([
```

Comment:
Was this the automatic linting?

Reply:
There were additional changes as well -- we had to add an extra method into the parent animation producer class so that empty arrays of runners won't be added to the timeline (resulted in a bug because constructTimeline method assumed every array had at least one element).